### PR TITLE
Revert wework/plasma#248

### PR DESCRIPTION
**Pull request by kobim**
Restores the css files of the DatePicker and ReactSelect (as Spacestation relies on them)

### CODE REVIEW

File: stories/DatePicker.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// import React from 'react';
+// import moment from 'moment';
+// import { storiesOf, action, linkTo } from '@kadira/storybook';
+// import DatePicker from '../src/components/DatePicker/DatePicker';
+
+// storiesOf('DatePicker', module)
+//   .add('primary', () => (
+//     <DatePicker selected={moment()} onChange={() => alert('changed')} />
+//   ));
```
Comment (maniator) [inline stories/DatePicker.js]: we can remove this file

File: src/index.js [hunk 2]
```diff
@@ -126,6 +132,7 @@ export {
   NumberInput,
   Loader,
   OverflowMenu,
+  // DatePicker,
```
Comment (maniator) [inline src/index.js]: we can remove the comments in this file

File: src/index.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ import OldSideNavBar from './components/SideNavBar/OldSideNavBar';
 import SideNavBarItem from './components/SideNavBar/OldSideNavBarItem';
 import SideNavBar from './components/SideNavBar/SideNavBar';
 import SideNavBarTop from './components/SideNavBar/SideNavBarTop';
+// import DatePicker from './components/DatePicker/DatePicker';
```
Comment (maniator) [inline src/index.js]: we can remove this file `components/DatePicker/DatePicker.jsx`

File: src/external/index.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+// import reactselect from './reactselect.scss';
```
Comment (maniator) [inline src/external/index.js]: we can remove this comment and this file

**Comment by kangax** [comment]
I wanted to try bringing datepicker styles to SS to see if it fixes style issues but whatever. 

Feel free to revert.

**Comment by kobim** [comment]
@kangax [tried that](https://github.com/WeConnect/spacestation-web/commit/0ff8f12c5a0094f85f2dfe022240a25391c1ae32)

**Comment by maniator** [comment]
> I wanted to try bringing datepicker styles to SS to see if it fixes style issues but whatever. Feel free to revert.
> […](#)
> Sent from my iPhone
> On Jul 17, 2019, at 10:16, Neal Lubin ***@***.***> wrote: @maniator commented on this pull request. In src/index.js: > @@ -126,6 +132,7 @@ export { NumberInput, Loader, OverflowMenu, + // DatePicker, we can remove the comments in this file — You are receiving this because your review was requested. Reply to this email directly, view it on GitHub, or mute the thread.

Let's do that as a followup to this.

This is blocking sprint work for MiMo

**Pull request merged**